apps/super-app/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
```

Comment:
Will these two deps be used? Don't appear to be imported anywhere

Reply:
They're peer dependencies of @mui/material, see https://mui.com/material-ui/getting-started/installation/